Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
Two things:

1. I whipped up a test disc using iDVD, saved it as a file system, and tarred it up (compressed with xz). The resulting tarball is 68 MB.
2. I can make a simple test for it, but it fails with "Floating point exception: 8". That seems like a problem much deeper than a build issue. Or it could just mean that vobcopy can't handle anything iDVD produces. :cry: